languages/php/src/BitwardenSDK.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+use Bitwarden\Sdk\Schemas\AccessTokenLoginRequest;
+use Bitwarden\Sdk\schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\Command;
+use FFI;
+use Swaggest\JsonDiff\Exception;
+
+
+class BitwardenSDK
+{
+    private BitwardenLib $bitwarden_lib;
+
+    private ClientSettings $clientSettings;
+
+    public ProjectsClient $projectsClient;
+
+    public SecretsClient $secretsClient;
+
+    private CommandRunner $commandRunner;
+
+    private FFI\CData $handle;
+
+    public function __construct()
+    {
+        $this->clientSettings = new ClientSettings();
+        $this->clientSettings->apiUrl = getenv('API_URL') ?: 'https://api.bitwarden.com';
+        $this->clientSettings->identityUrl = getenv('IDENTITY_URL') ?: 'https://identity.bitwarden.com';
+        $this->clientSettings->userAgent = getenv('USER_AGENT') ?: 'SDK';
+        $this->clientSettings->deviceType = getenv('DEVICE_TYPE') ?: 'SDK';
+
+        $this->bitwarden_lib = new BitwardenLib();
+        $this->handle = $this->bitwarden_lib->init($this->clientSettings);
+
+        $this->commandRunner = new CommandRunner($this->bitwarden_lib, $this->handle);
+        $this->projectsClient = new ProjectsClient($this->commandRunner);
+        $this->secretsClient = new SecretsClient($this->commandRunner);
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function authorize(string $access_token)
```

Comment:
Can we rename this to `accessTokenLogin` to keep the API as close to the other bindings?

Reply:
renamed

Reply:
Sorry, maybe I wasn't completely clear, I meant the function name, so something like this (had to check that PHP uses underscores for function names, so we should follow that pattern):
```php
    public function access_token_login(string $access_token)
```